trl/trainer/dpo_trainer.py

```diff
@@ -162,6 +164,7 @@ def __init__(
         generate_during_eval: bool = False,
         compute_metrics: Optional[Callable[[EvalLoopOutput], Dict]] = None,
         precompute_ref_log_probs: bool = False,
+        dataset_num_proc = None,
```

Comment:
```suggestion
        dataset_num_proc: int = 2,
```

Reply:
thats reasonable

Reply:
Please see my latest comment, I think it is good to make it configurable and set it to None by default